kubernetes-manifests/adservice.yaml

```diff
@@ -34,6 +34,10 @@ spec:
         env:
         - name: PORT
           value: "9555"
+        # - name: DISABLE_STATS
+        #   value: 1
```

Comment:
sadly these `1`s in YAML need to be `"1"` otherwise the validation will fail AFAIK.

Reply:
Hmm ok, I think I was able to "kubectl apply" these, but quoted them anyway.